source/install/config-ssl-http2-apache2.rst

```diff
@@ -21,30 +21,30 @@ When opened, edit it to look something like the following:
 
 		# setup the proxy
 		<Proxy *>
-		  Order allow,deny
-		  Allow from all
+			Order allow,deny
+			Allow from all
 		</Proxy>
 
 		RewriteEngine On
-		RewriteCond %{REQUEST_URI} ^/api/v3/users/websocket [NC,OR]
+		RewriteCond %{REQUEST_URI} /api/v[0-9]+/(users/)?websocket [NC,OR]
 		RewriteCond %{HTTP:UPGRADE} ^WebSocket$ [NC,OR]
 		RewriteCond %{HTTP:CONNECTION} ^Upgrade$ [NC]
 		RewriteRule .* ws://127.0.0.1:8065%{REQUEST_URI} [P,QSA,L]
 		RewriteCond %{DOCUMENT_ROOT}/%{REQUEST_FILENAME} !-f
 		RewriteRule .* http://127.0.0.1:8065%{REQUEST_URI} [P,QSA,L]
 
-		<Location /api/v3/users/websocket>
-		  Require all granted
-		  ProxyPass ws://127.0.0.1:8065/api/v3/users/websocket
-		  ProxyPassReverse ws://127.0.0.1:8065/api/v3/users/websocket
-		  ProxyPassReverseCookieDomain 127.0.0.1 mysubdomain.mydomain.com
-		</Location>
+		<LocationMatch "^/api/v(?<apiversion>[0-9]+)/(?<apiusers>users/)?websocket">
+        		Require all granted
+        		ProxyPass ws://127.0.0.1:8065/api/v%{env:MATCH_APIVERSION}/%{env:MATCH_APIUSERS}websocket
+        		ProxyPassReverse ws://127.0.0.1:8065/api/v%{env:MATCH_APIVERSION}/%{env:MATCH_APIUSERS}websocket
```

Comment:
Should it have `/` before `websocket` like `{env:MATCH_APIUSERS}/websocket`?

Do we have somewhere in our docs where we initially define how/what to setup with `MATCH_APIUSERS` and `MATCH_APIVERSION`? If none, then we might need to define it first.

Reply:
@saturninoabril These variable aren't something the user needs to setup, they are auto generated by the expression in the `LocationMatch` directive. Also the slash is part of the match there, so if that segment exists in the requested path the segment (with trailing slash) will be placed in the proxy URL as well.